### PR TITLE
Announce students for GSoC 2015

### PR DESCRIPTION
**Pull request by r-gaia-cs**
@adina Can you review this?

### CODE REVIEW

File: blog/2015/05/gsoc-projects-2015.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  The projects are (order alphabetically by student's last name):
+</p>
+<table class="table table-striped">
+  <tr>
+    <th>Student</th>
+    <th>Project</th>
+    <th>Mentor</th>
+    <th>Links</th>
+  <tr>
+    <td>Piotr Banaszkie</td>
```
Comment (pbanaszkiewicz) [inline blog/2015/05/gsoc-projects-2015.html]: Missing part of my last name: "Banaszkiewicz".

File: blog/2015/05/gsoc-projects-2015.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+author: Raniere Silva
+title: "GSoC Projects for 2015"
+date: 2015-05-01
+time: "20:00:00"
+category: ["Announcements"]
+---
+<!-- start excerpt -->
+<p>
+  We're very pleased to announce that three students will be working on
+  <a href="https://www.google-melange.com/gsoc/homepage/google/gsoc2015">Google Summer of Code</a>
```
Comment (gvwilson) [inline blog/2015/05/gsoc-projects-2015.html]: add "projects" at the end of this line

File: blog/2015/05/gsoc-projects-2015.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  under <a href="http://numfocus.org/">NumFOCUS</a> umbrella that we are helping coordinating.
```
Comment (gvwilson) [inline blog/2015/05/gsoc-projects-2015.html]: under _the_ NumFOCUS umbrella that we are helping to coordinate.

File: blog/2015/05/gsoc-projects-2015.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+</p>
+<!-- end excerpt -->
+<p>
+  The projects are (order alphabetically by student's last name):
```
Comment (gvwilson) [inline blog/2015/05/gsoc-projects-2015.html]: Just "The projects are:" - no need to say how it's ordered.

File: blog/2015/05/gsoc-projects-2015.html [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+</p>
+<table class="table table-striped">
+  <tr>
+    <th>Student</th>
+    <th>Project</th>
+    <th>Mentor</th>
+    <th>Links</th>
+  <tr>
+    <td>Piotr Banaszkie</td>
+    <td>Enhance Amy, a workshop-management platform for Software Carpentry</td>
```
Comment (gvwilson) [inline blog/2015/05/gsoc-projects-2015.html]: turn "Amy" into a link to https://github.com/swcarpentry/amy

File: blog/2015/05/gsoc-projects-2015.html [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+    <td>Alexander Croy</td>
+    <td>
+      <a href="https://github.com/numfocus/gsoc/blob/master/2015/proposals/Amit.md">Proposal</a>,
+      <a href="https://juliaquantum.github.io/news/index.html">blog</a>
+      (of JuliaQuantum).
+    </td>
+  </tr>
+</table>
+<p>
+  Many others open source projects are participating in Google Summer of Code
```
Comment (gvwilson) [inline blog/2015/05/gsoc-projects-2015.html]: GSoC instead of Google Summer of Code

Comment (gvwilson) [inline blog/2015/05/gsoc-projects-2015.html]: open source scientific computing projects, and then delete the next line

File: blog/2015/05/gsoc-projects-2015.html [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+  <li><a href="https://www.google-melange.com/gsoc/org2/google/gsoc2015/scilab_enterprises">Scilab Enterprises</a>,</li>
+  <li><a href="https://www.google-melange.com/gsoc/org2/google/gsoc2015/lmonade">lmonade</a> and others.</li>
+</ul>
+<p>
+  We want to thanks all the students that apply to Google Summer of Code,
+  was very hard to select the students for the limited number of slots,
+  and everyone that signed to be a mentor.
+</p>
+<p>
+  It's going to be an exciting summer...
```
Comment (gvwilson) [inline blog/2015/05/gsoc-projects-2015.html]: Use this to close the preceding paragraph.

**Comment by r-gaia-cs** [comment]
@pbanaszkiewicz and @gvwilson Corrections done.

**Pull request merged**